templates/single/quiz/parts/choice-box.php

```diff
@@ -42,6 +42,14 @@
 	$class = 'tutor-quiz-multiple-variation';
 }
 //phpcs:enable WordPress.WP.GlobalVariablesOverride.Prohibited
+
+/**
+ * Type single_choice removed. It's handled by multiple_choice type.
+ *
+ * @since 3.0.0
+ */
+$multiple_answer_allowed = isset( $question_settings['has_multiple_correct_answer'] ) && '1' === $question_settings['has_multiple_correct_answer'];
```

Comment:
Since we making type check here
`'1' === $question_settings['has_multiple_correct_answer']`
make sure `$question_settings['has_multiple_correct_answer']` is int